src/scripts/components/globe-navigation/globe-navigation.styl

```diff
@@ -7,13 +7,18 @@
   z-index: 1
   display: flex
 
-.compassIcon
-  padding-right: emCalc(16px)
-  padding-left: emCalc(25px)
+.compass
+  margin-right: emCalc(16px)
+  margin-left: emCalc(25px)
```

Comment:
Why do we need these strange margins? Can we solve this in a better way with flexbox?